s3-eventbridge-wildcard-sam/example-pattern.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "Wildcard pattern match in Amazon EventBridge rule",
+  "description": "The SAM template deploys an Amazon S3 bucket, an Amazon EventBridge rule with wildcard pattern match and an Amazon EventBridge target as Amazon SQS Queue.", 
```

Comment:
Please shorten it to 150 characters or less

Reply:
Fixed.